.github/workflows/ci_cd.yml

```diff
@@ -152,6 +152,8 @@ jobs:
 
           echo "stable_container=${{ env.DOCKER_PACKAGE }}:${{ env.LATEST_STABLE_DOCKER_IMAGE_VERSION }}" >> $GITHUB_OUTPUT
 
+          cat $GITHUB_OUTPUT
```

Comment:
@RobPasMue  I feel this is convenient in case we want to see what image is being used. Since it does not contain any secret, do you think its okay?

Reply:
Fine